[IMP] CRM: Sales team lead assignment clarification (#18332)
odoo/documentation: content/applications/sales/crm/pipeline/manage_sales_teams.rst

:guilabel:`Save & Close` button to finish, or click :guilabel:`Save & New` to open a blank form and
keep adding sales team members.

.. note::

>    When using rule-based assignment, Odoo automatically limits the maximum number of leads that an

Reviewer: ```suggestion
   When using rule-based assignment, Odoo automatically limits the maximum number of leads an
```

>    individual teammember can be be assigned in a day. This limit is determined by the number in the

Reviewer: ```suggestion
   individual team member can be assigned in a day. This limit is determined by the number in the
```

   :guilabel:`Capacity` field and is equal to that number divided by 30 (for the number of days in

>    the period.) **This may result in leads going unassigned if all sales team member's daily

Author: ```suggestion
   the period.) **This may result in leads going unassigned if all sales team members' daily
```

   capacity has been met.**

   For example, Sales Team Member John Doe has a :guilabel:`Capacity` maximum of 120 leads in a
   30-day period. This means that the database automatically assigns John a maximum of 4 leads per
   day. If John were the only member of the sales team and 10 leads came in in a single day, only 4

>    would be assigned to him. The other 6 would go unassigned.

Reviewer 2: ```suggestion
.. note::
   When using rules-based assignment, Odoo automatically limits the maximum number of leads an
   individual team member can be assigned in a day. This limit is determined by the number in the
   :guilabel:`Capacity` field and is equal to that number divided by 30 (for the number of days in
   the period). **This may result in leads going unassigned if all sales team members' daily
   capacity has been met.**

.. example::
   Sales team member John Doe has a :guilabel:`Capacity` maximum of 120 leads in a 30-day period.
   This means that the database automatically assigns John a maximum of 4 leads per day. If John
   were the only member of the sales team and 10 leads came in on a single day, only 4 would be
   assigned to him. The other 6 would go unassigned.
```
This block contains conceptual information about the daily cap and a concrete "John Doe" example.

maybe we can split this into a `.. note::` for the daily cap explanation and a `.. example::` for the John Doe scenario. Note blocks are for important additional information and example is for showing examples.

Also grabbed a few small things:
- "rule-based" → "rules-based": the doc uses "rules-based" everywhere else
- "Sales Team Member" → "sales team member": not a proper noun
- "came in in" → "came in on": typo?

Author: Good note! I was trying to avoid having back-to-back admonitions, but I think it's probably worth having the separation here now that you point it out.

Author: Tweaked the language around "came in on" to be less ambiguous for any future translations.
